recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -17,10 +17,9 @@ python () {
 # Enable Etnaviv and Freedreno support
 PACKAGECONFIG_append_use-mainline-bsp = " gallium etnaviv kmsro freedreno"
 
-# Enable OSMesa which also requires dri (classic) swrast
-PACKAGECONFIG_remove_use-nxp-bsp = "gallium"
-PACKAGECONFIG_append_use-nxp-bsp = " osmesa"
-DRIDRIVERS_use-nxp-bsp = "swrast"
+# For NXP BSP, enable OSMesa for parts with DRM
+PACKAGECONFIG_remove_use-nxp-bsp_imxdrm = "gallium"
+PACKAGECONFIG_append_use-nxp-bsp_imxdrm = " osmesa"
```

Comment:
Hi @zandrey, I believe you added `swrast` in the first place. Were you having a build issue without it? Or was it a runtime issue? Prabhu says we should not be using `swrast` when we have hardware acceleration available.

Reply:
@thochstein IIRC, I had a build issue back then when HW acceleration was not available in Mesa (not implemented).

If the HW acceleration support is already fully available in Mesa - then I guess this can be dropped.

-- andrey